src/mlt++/MltProperties.cpp

```diff
@@ -431,3 +431,18 @@ Animation *Properties::get_anim(const char *name)
 {
 	return new Animation( mlt_properties_get_animation( get_properties(), name ) );
 }
+
+int Properties::set( const char *name, Properties& properties )
+{
+	return mlt_properties_set_properties( get_properties(), name, properties.get_properties() );
+}
+
+Properties* Properties::get_props( const char *name )
```

Comment:
I use "get_props" here because there is a name collision when Mlt::Service inherits Mlt::Properties. Mlt::Service already has a method named "get_properties". I'm open to suggestions here.

Reply:
But this method has a different signature and the one in Mlt::Service can overload this one if named the same?

Reply:
That is also my understanding. I couldn't immediately get the compiler to find the child function. But the two functions have nothing in common except the name. So I am also concerned that the overloaded names might cause confusion for other developers. I can keep working to figure out the compiler issue if you prefer the overloaded function names.

Reply:
I agree it is not an appropriate usage of overloading. I am fine with the names you have used.